staging/src/k8s.io/api/admission/v1beta1/types.go

```diff
@@ -75,6 +75,9 @@ type AdmissionRequest struct {
 	// Defaults to false.
 	// +optional
 	DryRun *bool `json:"dryRun,omitempty" protobuf:"varint,11,opt,name=dryRun"`
+	// Options is the operation option structure user passed to the API call. e.g. `DeleteOption` or `CreateOption`.
+	// +optional
+	Options runtime.RawExtension `json:"options,omitempty" protobuf:"bytes,12,opt,name=options"`
```

Comment:
omitempty does not work for RawExtensions. This will render as `null`

Reply:
Good to know! Should I remove it from the above `Object` and `OldObject` declarations above that are also `runtime.RawExtension`? I can't think of any reason not to, but maybe it would have a backward incompatible impact?

Reply:
I don't feel strongly, but I'd lean toward making this consistent with Object and OldObject.

Reply:
I'll leave it in for consistency.